airlock/templates/file_browser/index.html

```diff
@@ -1,9 +1,34 @@
 {% extends "base.html" %}
 
-{% block content %}
-
-{% #article_header title="File Browser" %}
-{% /article_header %}
+{% block full_width_content %}
+
+{% fragment as action_button %}
+  {% if is_author %}
+    <form action="" method="POST">
+    {% csrf_token %}
+    <input type=hidden name="workspace" value="{{ workspace.name }}"/>
+    <input type=hidden name="output_request" value="{{ output_request.request_id }}"/>
+    {% #button type="submit" tooltip="This output request is ready to be reviewed" variant="success" %}Submit Output Request For Review{% /button %}
+    </form>
+  {% elif is_output_checker %}
+    <form action="" method="POST">
+    {% csrf_token %}
+    <input type=hidden name="workspace" value="{{ workspace.name }}"/>
+    <input type=hidden name="output_request" value="{{ output_request.request_id }}"/>
+    {% #button type="submit" tooltip="Release these to jobs.opensafely.org" variant="danger" %}Release Files{% /button %}
```

Comment:
nit (for later): hovering over the tooltip makes a scrollbar appear, and the page shift to the right which is a bit weird

Reply:
interesting. One for @tomodwyer input, perhaps

Reply:
Yeah this is a really annoying bug, the only solution I've found is to add custom classes to move the tooltip in a different direction 😢